src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/BitsHelper.java

```diff
@@ -0,0 +1,448 @@
+
+    private final Map<String, Integer> catSacks = Util.make(new HashMap<>(), map -> {
+        map.put("POCKET_SACK_IN_A_SACK", 8000);
+        map.put("LARGE_DUNGEON_SACK", 14000);   //sacks can't be traded, but I will add them anyway in case they will be auctionable at some point
+        map.put("RUNE_SACK", 14000);
+        map.put("FLOWER_SACK", 14000);
+        map.put("DWARVEN_MINES_SACK", 14000);
+        map.put("CRYSTAL_HOLLOWS_SACK", 14000);
+    });
+
+    private final Map<String, Integer> catAbiphone = Util.make(new HashMap<>(), map -> {
+        map.put("TRIO_CONTACTS_ADDON", 6450);
+        map.put("ABICASE_SUMSUNG_1", 15000); // Original skibidiblock ID is all "ABICASE" with "Model" extra data. Admins why
+        map.put("ABICASE_SUMSUNG_2", 25000);
+        map.put("ABICASE_REZAR", 26000);
+        map.put("ABICASE_BLUE_AQUA", 17000);
+        map.put("ABICASE_BLUE_BLUE", 17000);
+        map.put("ABICASE_BLUE_GREEN", 17000);
+        map.put("ABICASE_BLUE_RED", 17000);
+        map.put("ABICASE_BLUE_YELLOW", 17000);
+    });
+
+    private final Map<String, Integer> catDyes = Util.make(new HashMap<>(), map -> {
+        map.put("DYE_PURE_WHITE", 250000);
+        map.put("DYE_PURE_BLACK", 250000);
+    });
+
+    private final Map<String, Integer> catEnchants = Util.make(new HashMap<>(), map -> {
+        map.put("ENCHANTMENT_HECATOMB_1", 6000);
+        map.put("ENCHANTMENT_EXPERTISE_1", 4000);
+        map.put("ENCHANTMENT_COMPACT_1", 4000);
+        map.put("ENCHANTMENT_CULTIVATING_1", 4000);
+        map.put("ENCHANTMENT_CHAMPION_1", 4000);
+        map.put("ENCHANTMENT_TOXOPHILITE_1", 4000);
+    });
+
+    private final Map<String, Integer> catEnrichments = Util.make(new HashMap<>(), map -> {
+        map.put("TALISMAN_ENRICHMENT_SWAPPER", 200);
+        map.put("TALISMAN_ENRICHMENT_WALK_SPEED", 5000);
+        map.put("TALISMAN_ENRICHMENT_INTELLIGENCE", 5000);
+        map.put("TALISMAN_ENRICHMENT_CRITICAL_DAMAGE", 5000);
+        map.put("TALISMAN_ENRICHMENT_CRITICAL_CHANCE", 5000);
+        map.put("TALISMAN_ENRICHMENT_STRENGTH", 5000);
+        map.put("TALISMAN_ENRICHMENT_DEFENSE", 5000);
+        map.put("TALISMAN_ENRICHMENT_HEALTH", 5000);
+        map.put("TALISMAN_ENRICHMENT_MAGIC_FIND", 5000);
+        map.put("TALISMAN_ENRICHMENT_FEROCITY", 5000);
+        map.put("TALISMAN_ENRICHMENT_SEA_CREATURE_CHANCE", 5000);
+        map.put("TALISMAN_ENRICHMENT_ATTACK_SPEED", 5000);
+    });
+
+    private final Map<String, Map<String, Integer>> categories = Util.make(new HashMap<>(), map -> {
+        // Категории
+        map.put("Kat Items", catKat);
+        map.put("Upgrade Components", catUpgradeComponents);
+        map.put("Sacks", catSacks);
+        map.put("Abiphone Supershop", catAbiphone);
+        map.put("Dyes", catDyes);
+        map.put("Stacking Enchants", catEnchants);
+        map.put("Enrichments", catEnrichments);
+    });
+
+    // I'm not sorry
+    private static class BestItemsResult {
```

Comment:
I didn't really pay attention to the usages of this but if these fields aren't mutated outside of the constructor you can use a record instead.